logprep/abc/input.py

```diff
@@ -312,9 +318,9 @@ def _add_env_enrichment_to_event(self, event: dict):
 
     def _add_arrival_time_information_to_event(self, event: dict):
         new_field = {
-            self._config.preprocessing.get(
-                "log_arrival_time_target_field"
-            ): TimeParser.now().isoformat()
+            self._config.preprocessing.get("log_arrival_time_target_field"): TimeParser.now(
+                self._log_arrival_timestamp_timezone
+            ).isoformat()
         }
         add_fields_to(event, new_field)
```

Comment:
maybe we can restructure this a bit, just so that it is easier to read:
```suggestion
        target = self._config.preprocessing.get("log_arrival_time_target_field")
        time = TimeParser.now(self._log_arrival_timestamp_timezone).isoformat() 
        add_fields_to(event, {target: time})
```